src/components/account/company/appsco-account-devices.html

```diff
@@ -0,0 +1,232 @@
+                @apply(--layout-vertical);
+                @apply(--appsco-devices);
+            }
+            :host appsco-account-devices-item:first-of-type {
+                border-top: none;
+            }
+            :host .message {
+                color: var(--secondary-text-color);
+                font-style: italic;
+                @apply(--paper-font-body2);
+                @apply(--info-message);
+            }
+            :host paper-progress {
+                width: 100%;
+                position: absolute;
+                top: 0;
+                left: 0;
+                @apply(--account-devices-paper-progress);
+            }
+            .info-total {
+                margin-top: 10px;
+            }
+            .total {
+                @apply(--paper-font-caption);
+                color: var(--secondary-text-color);
+            }
+
+        </style>
+
+        <iron-ajax
+                id="ironAjaxGetDevices"
+                url="[[ devicesApi ]]"
+                on-response="_onDeviceResponse"
+                on-error="_handleError"
+                headers="{{ _computedHeaders }}"></iron-ajax>
+
+        <paper-progress id="progress" indeterminate></paper-progress>
+
+        <template is="dom-if" if="{{ !preview }}">
+            <div class="devices">
+                <template is="dom-repeat" items="[[ _devices ]]" rendered-item-count="{{ renderedCount }}">
+                    <appsco-account-devices-item
+                            device="[[ item ]]"
+                            account="[[ account ]]"
+                            preview = "[[ preview ]]">
+                    </appsco-account-devices-item>
+                </template>
+            </div>
+        </template>
+
+        <template is="dom-if" if="{{ _message }}">
+            <p class="message">
+                [[ _message ]]
+            </p>
+        </template>
+
+        <div class="info-total">
+            <div class="total">
+                Total registered devices: [[ _totalDevices ]]
+            </div>
+        </div>
+
+    </template>
+
+    <script>
+        Polymer({
+
+            is: 'appsco-account-devices',
+
+            properties: {
+
+                /**
+                 * Account
+                 */
+                account: {
+                    type: Object,
+                    value: function () {
+                        return {};
+                    }
+                },
+
+                /**
+                 * Get role devices link.
+                 */
+                devicesApi: {
+                    type: String
+                },
+
+                /**
+                 * Number of devices to load.
+                 */
+                size: {
+                    type: Number
+                },
+
+                /**
+                 * Indicates if device should be in preview mode rather then full detailed view.
+                 */
+                preview: {
```

Comment:
Is this used?

Reply:
It's used in 'appsco-manage-account-components-page' component, where only device number should be displayed, not concrete devices list.